docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -0,0 +1,318 @@
+---
+# required/recommended metadata
+
+title: Datetime and DateTimeOffset Support in System.Text.Json
+description: An overview of how Datetime and DateTimeOffset types are supported in the System.Text.Json library.
+author: layomia
+ms.author: laakinri
+ms.date: 07/22/2019
+ms.topic: [TOPIC TYPE]
+ms.prod: [PRODUCT VALUE]
+helpviewer_keywords:
+  - "JSON DateTime JSON DateTimeOffset"
+  - "Serializer, Utf8"
+  - "JSON Reader, JSON Writer, times"
+
+---
+# Datetime and DateTimeOffset Support in System.Text.Json
+
+The <xref:System.Text.Json.JsonSerializer>, <xref:System.Text.Json.Utf8JsonReader>, <xref:System.Text.Json.Utf8JsonWriter>, and <xref:System.Text.Json.JsonElement> parse and write <xref:System.DateTime> and <xref:System.DateTimeOffset> text representations according to an extended profile of the ISO 8601-1:2019 format.
+
+## Date and Time Components
+
+ISO 8601-1:2019 defines the following components for date and time text representations:
```

Comment:
Is there somewhere you can link to that already has this information?

Reply:
We can link to the place to buy the spec, but you can't see the content without $.

Reply:
Ok. IMO this article could be split into two. One article that shows how to use DateTime and DateTimeOffset with the serializer, reader, writer, document model. And how to write JsonConverters to change how dates a processed. And another article that goes in-depth with ISO 8061 and how DateTime/DateTimeOffset are mapped to it.

If you don't want to do that then at least move the spec heavy discussion to the bottom of the article.